<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="General">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>file</string>
       </property>
       <widget class="QLabel" name="countInfo">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>42</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>模型点数：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="count">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>42</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="expressionDirInfo">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>94</y>
          <width>108</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>表情模型输出路径：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="expressionDir">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>94</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="expressionListInfo">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>68</y>
          <width>108</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>表情名称输出路径：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="expressionList">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>68</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="expressionDirInfo_2">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>120</y>
          <width>78</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>blend shape：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="blendShapeName">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>120</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="rangeMin">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>146</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="countInfo_2">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>146</y>
          <width>96</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>表情参数最小值：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rangeMax">
        <property name="geometry">
         <rect>
          <x>135</x>
          <y>172</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="countInfo_3">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>172</y>
          <width>96</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>表情参数最大值：</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
